app/components/content-and-image-box/content-and-image-box.tsx

```diff
@@ -87,16 +91,18 @@ export const ContentAndImageBox: React.FC<Props> = ({
       </div>
 
       <div
-        className={classNames(
-          "relative aspect-square max-h-60 bg-white shadow-xl md:h-[30vw] md:max-h-96",
-          {
-            "md:-ml-[100px]": direction === "left",
-            "md:-mr-[100px]": direction === "right",
-          },
-        )}
+        className={classNames("relative bg-white shadow-xl md:h-[30vw]", {
+          "md:-ml-[100px]": direction === "left",
+          "md:-mr-[100px]": direction === "right",
+        })}
       >
-        <Pattern {...randomPattern(direction, patternClassName)} />
-        {image}
+        {isHydrated && (
+          <Pattern {...randomPattern(direction, patternClassName)} />
+        )}
```

Comment:
Ja såklart 💡 Det er jo den enkle måten å løse dette problemet!

Jeg har grubblet litt og sett litt på koden på hvordan vi kunne få gjøre randomnessen deterministisk. Men slik patterns er satt opp er det litt av spageti av tilfeldighet.

Såklart mye bedre å bare droppe SSR for patterns, er heller ikke en nødvendighet for brukerene.
💯 👏